server/persistence/persistence_test.go

```diff
@@ -39,6 +41,24 @@ func TestInMemoryDatabase(t *testing.T) {
 	db := persistence.NewInMemoryDB()
 	testDb(t, db)
 }
+func TestTelemetryTable(t *testing.T) {
+
+	const DB_FILENAME = "tmp.db"
```

Comment:
I am thinking the in memory db is more appropriate for this test case, but not a big deal.

Reply:
Updated

Reply:
Maybe also change the name of the file? Since you switched from persistence to `inMemoryDB` ?

Reply:
The file name is for the overall database package, which is called persistence so it matches the package..